osbs/api.py

```diff
@@ -799,6 +830,81 @@ def create_build(self, **kwargs):
         """
         return self._do_create_prod_build(**kwargs)
 
+    @osbsapi
+    def create_source_container_build(
+        self,
+        sources_for_koji_build_nvr=None,
```

Comment:
Maybe this is fine exactly as it is even though the Koji task should also accept a Koji build ID: can koji-containerbuild look up the n-v-r for us and pass it in?

Reply:
I was thinking about doing lookup for NVR from ID on osbs-client or koji-container build side, so fine by me :)

Reply:
what about 'sources_for_koji_build_id' ? koji-c will get both nvr & id if provided, so we should get both

Reply:
Was agreed that I'll add `sources_for_koji_build_id` later if needed. So far I don;t have usage for that I need N-V-R